pool/src/auctions/bad_debt_auction.rs

```diff
@@ -48,15 +48,11 @@ pub fn create_bad_debt_auction_data(e: &Env, backstop: &Address) -> AuctionData
     let backstop_token = backstop_client.backstop_token();
     let pool_backstop_data = backstop_client.pool_data(&e.current_contract_address());
     let usdc_token = storage::get_usdc_token(e);
-    let blnd_token = storage::get_blnd_token(e);
     let usdc_to_base = pool.load_price(e, &usdc_token);
-    let blnd_to_base = pool.load_price(e, &blnd_token);
```

Comment:
Can we drop the `BLND` oracle values in the unit tests in this file?

This ensures if we ever introduce a dependency on an oracle price forBLND it will cause tests to clearly fail.